schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "xyz",
+    "version": "0.0.1",
```

Comment:
Not sure how best/simplest to wire the version through an example like this.  Ideally we would define the version once (maybe just here?) and it would get injected in everywhere else.  Perhaps the provider itself should load the schema and use the version (and do dispatch based on resources listed in it?). 

 It also unfortunately doesn't make it into the `package.json` from here as noted below.

Reply:
We'll keep this as is for now.